src/3p.js

```diff
@@ -109,18 +109,25 @@ function executeAfterWriteScript(win, fn) {
  * @param {string} src
  */
 export function validateSrcPrefix(prefix, src) {
-
+  const validPrefixes = [];
   if (!isArray(prefix)) {
     prefix = [prefix];
   }
-
-  for (const p of prefix) {
-    if (src.indexOf(p) === 0) {
+  const totalPrefixes = prefix.length;
+  if (src !== undefined) {
+    let p = 0;
```

Comment:
if we inline P we get an error on the Build about p being read-only, See Build No. 3858 https://travis-ci.org/ampproject/amphtml/builds/109090168

<img width="1164" alt="screen shot 2016-02-14 at 12 20 11 pm" src="https://cloud.githubusercontent.com/assets/1742233/13035127/06c509c8-d316-11e5-8575-f38df774ed69.png">


I can still inline this if you would like to see if the build passes.